quiz.js

```diff
@@ -17,9 +18,24 @@ db.dropDatabase();
 
 // your code begins here
 
+db.createCollection("houses");
```

Comment:
Also not necessary. In Mongo, if you try to call a collection that doesn't exist, it will create that collection for you on the fly. The only reason to use `createCollection` is if you want to define specific properties for the collection (such as maximum file size, or maximum number of items).